src/operation/get-response-as-json-schema.ts

```diff
@@ -11,27 +16,38 @@ const { json: isJSON } = require('../lib/matches-mimetype').default;
  * @param response Response object to build a JSON Schema object for its headers for.
  * @returns object
  */
-function buildHeadersSchema(response) {
+function buildHeadersSchema(response: ResponseObject) {
   const headers = response.headers;
 
-  const headersSchema = {
+  const headersSchema: SchemaObject = {
     type: 'object',
     properties: {},
   };
 
   Object.keys(headers).forEach(key => {
-    if (headers[key] && headers[key].schema) {
+    if (headers[key] && (headers[key] as OpenAPIV3.HeaderObject | OpenAPIV3_1.HeaderObject).schema) {
```

Comment:
Might want to make a `HeaderObject` in `rmoas.types` that shortcuts having to repeat `OpenAPIV3.HeaderObject | OpenAPIV3_1.HeaderObject`.